k8s/base/exporter.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: basin-exporter
+  labels:
+    app: basin-exporter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: basin-exporter-v1
+  template:
+    metadata:
+      labels:
+        app: basin-exporter-v1
+    spec:
+      containers:
+        - name: basin-exporter
+          image: textile/basin_exporter
+          imagePullPolicy: Always
+          command: ["/bin/app"]
+          args: [
+            "--export-bucket", "$(EXPORT_BUCKET)",
+            "--database-url", "$(DATABASE_URL)",
+            "--export-credentials", "$(EXPORT_CREDENTIALS)",
+            "--bind-health-address", "$(BIND_HEALTH_ADDRESS)",
+           #"--export-schedule", "$(EXPORT_SCHEDULE)",
```

Comment:
this is commented on because i could not figure out a syntax in the env file that worked. it's just using the default